vali_objects/position.py

```diff
@@ -519,14 +528,27 @@ def _clamp_and_validate_leverage(self, order) -> bool:
         is_first_order = len(self.orders) == 0
         proposed_leverage = self.net_leverage + order.leverage
         min_position_leverage, max_position_leverage = leverage_utils.get_position_leverage_bounds(self.trade_pair, order.processed_ms)
+
+        current_adjusted_leverage = abs(self.net_leverage) * self.trade_pair.leverage_multiplier
```

Comment:
if we build a system where you can submit multiple orders at the same time (ignore the current 10 second cooldown) would we have an issue here or would it just sequentially fill orders until the leverage limit is reached

Reply:
`add_order` will sequentially fill orders until the leverage limit is reached. That serves as the only way for miners to submit orders